lib/net.js

```diff
@@ -146,7 +146,7 @@ function Socket(options) {
   } else if (options.fd !== undefined) {
     this._handle = createHandle(options.fd);
     this._handle.open(options.fd);
-    if ((options.fd == 1 || options.fd == 2) &&
+    if ((options.fd === 1 || options.fd === 2) &&
```

Comment:
Well, you could just leave `==`. :-D

`options.fd = +options.fd` should make it a number too. There are also bit-shifting things I think that might be more efficient.

But ultimately, I'm not sure this needs to be changed at all.